rails_event_store/lib/rails_event_store/all.rb

```diff
@@ -20,6 +21,7 @@ module RailsEventStore
   InvalidHandler              = RubyEventStore::InvalidHandler
   InvalidPageStart            = RubyEventStore::InvalidPageStart
   InvalidPageSize             = RubyEventStore::InvalidPageSize
+  CorrelatedCommands          = RubyEventStore::CorrelatedCommands
```

Comment:
Why here? Seems not to be correlated with that PR :)

Reply:
I forgot to do it in the other PR :) it's slightly related via being about _correlation_ :)